array-methods.js

```diff
@@ -58,20 +104,52 @@ var stateSums = null;
     round this number to the nearest dollar before moving on.
   )
  */
-var sumOfHighInterests = null;
+
+
+var sumOfHighInterests = Object.entries(stateSums).filter(function (element) {
+  if (['WI', 'IL', 'WY', 'GA', 'DE', 'OH'].includes(element[0])) {
+    return false;
+  } else {
```

Comment:
since includes returns a true/false value, you could just return the result of the `.includes`